### PR TITLE
README: restructure for contributor onboarding

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@kinvolk.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,23 @@
-# Kinvolk Contribution Guidelines <img align="right" src="https://avatars2.githubusercontent.com/u/14073052?s=200&v=4" alt="Kinvolk" width="160">
+# Contributions with Kinvolk
 
-This repository contains information on contributing to Kinvolk projects.
+<img align="right" src="https://avatars2.githubusercontent.com/u/14073052?s=200&v=4" alt="Kinvolk" width="160">
+
+Here you will find information on contributing to Kinvolk projects, whether first time contribution, or referencing common guidlines (like the CoC).
```
Comment (joaquimrocha) [inline README.md:5]:
```suggestion
Here you will find information on contributing to Kinvolk projects, whether it's a first time contribution, or referencing common guidlines (like the CoC).
```

File: README.md [hunk 2]
```diff
@@ -14,16 +26,52 @@ its [Code of Conduct](./CODE_OF_CONDUCT.md) for all its projects and events.
 Please read and uphold this code-of-conduct while participating in Kinvolk
 projects and events.
 
-## Email and Chat
+## Projects
+
+There are many repos in the [Kinvolk](https://github.com/kinvolk/) org, though here is the highlights of projects.
```
Comment (joaquimrocha) [inline README.md:31]:
```suggestion
There are many repos in the [Kinvolk](https://github.com/kinvolk/) org. This section highlights the main projects.
```

File: README.md [hunk 3]
```diff
@@ -14,16 +26,52 @@ its [Code of Conduct](./CODE_OF_CONDUCT.md) for all its projects and events.
...
+
+### Flatcar Container Linux
+
+- [Flatcar repositories](https://github.com/search?q=org%3Akinvolk+topic%3Aflatcar&type=repositories)
+- [Flatcar docs](https://kinvolk.io/docs/flatcar-container-linux/latest)
+- [Flatcar issue tracker](https://github.com/kinvolk/flatcar/issues)
+- [contributing to Flatcar](https://github.com/kinvolk/Flatcar#contributing-to-flatcar-container-linux)
+- chat: [#flatcar](irc://irc.freenode.org:6667/#flatcar) IRC channel on freenode.org
+- mailing-list: [flatcar-user](https://groups.google.com/g/flatcar-linux-user)
```
Comment (joaquimrocha) [inline README.md:40]: Nitpick: I'd not repeat the name of the project since the context is clear, but it's done now so :shrug: .

**Review by joaquimrocha — APPROVED** [review]
Overall looks good to me. I added some suggestions for language that sounds clearer to me, but you're the native speaker :slightly_smiling_face:

File: README.md (hunk 3 above)
Comment (vbatts) [reply README.md:40]: I debated about that too :thinking:

File: README.md (hunk 1 above)
Comment (iaguis) [reply README.md:5]: s/guidlines/guidelines/

**Comment by vbatts** [comment]
Updated. PTAL.

**Review by joaquimrocha — APPROVED** [review]

**Pull request merged**